### PR TITLE
sparse-9: Add Android 9 generic sparse

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

File: sparse-9/etc/usb-moded/usb-moded-configfs.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[configfs]
```
Comment (sledges) [inline sparse-9/etc/usb-moded/usb-moded-configfs.ini:1]: This file is device-specific (e.g. Xperia 10 doesn't have this).

**Review by sledges — CHANGES_REQUESTED** [review]

**Comment by Thaodan** [comment]
@sledges Should we still merge this?

**Comment by sledges** [comment]
Well @jusa is still not convinced, although monich [said](https://github.com/mer-hybris/droid-hal-configs/pull/246#issuecomment-1025410915) that this does make certain amount of sense:) (as default/fallback set of configs for fresh ports).

IMHO, it doesn't affect/break Xperia 10 (ganges) which is on Android 9, and we won't have time to migrate it to sparse-9, but would certainly help for any new/old community android-9 ports coming in.

**Comment by Thaodan** [comment]
> Well @jusa is still not convinced, although monich
> [said](https://github.com/mer-hybris/droid-hal-configs/pull/246#issuecommen
> t-1025410915) that this does make certain amount of sense:) (as
> default/fallback set of configs for fresh ports).
That's the idea.
> IMHO, it doesn't affect/break Xperia 10 (ganges) which is on Android 9, and
> we won't have time to migrate it to sparse-9, but would certainly help for
> any new/old community android-9 ports coming in.
>

I pretty much agree we don't know what BSP a new port might use.

**Comment by saukko** [comment]
Agreed not to do this for now.

**Pull request closed without merging**